doc/user/bindings/cli.md

```diff
@@ -1212,6 +1212,74 @@ $ mlpack_hoeffding_tree --input_model_file tree.bin --test_file test_set.arff
  - [Mining High-Speed Data Streams (pdf)](http://dm.cs.washington.edu/papers/vfdt-kdd00.pdf)
  - [HoeffdingTree class documentation](../../user/methods/hoeffding_tree.md)
 
+## mlpack_image_converter
+{: #image_converter }
+
+#### Image Converter
+{: #image_converter_descr }
+
+```bash
+$ mlpack_image_converter [--channels 0] [--dataset_file <string>]
+        [--height 0] [--help] [--info <string>] --input [] [--quality 90]
+        [--save] [--verbose] [--version] [--width 0] [--output_file <string>]
+```
+
+A utility to load an image or set of images into a single dataset that can then be used by other mlpack methods and utilities. This can also unpack an image dataset into individual files, for instance after mlpack methods have been used. [Detailed documentation](#image_converter_detailed-documentation).
+
+
```

Comment:
```suggestion
```

Remove the extra empty lines.

Reply:
These are actually all generated, so there's probably an extra line added from the markdown binding generator.  I'll leave them as-is, otherwise they won't match what the `make markdown` target builds (the `./scripts/check-markdown-docs.sh` script will fail---which, actually, it didn't in CI, so I will need to look into that).